edx_sysadmin/api/views.py

```diff
@@ -64,19 +84,26 @@ def post(self, request):
                         msg = _("Triggered reloading branch: {} of repo: {}").format(
                             active_branch, repo_name
                         )
-                        logger.info(f"{self.__class__.__name__}:: {msg}")
```

Comment:
Is there a reason for dropping this log info message?

Reply:
Oh, was it moved down to `get_reload_response()` ? That's fine.

Reply:
Yes It's moved into a common method.